cpp/ql/lib/semmle/code/cpp/models/implementations/Memcpy.qll

```diff
@@ -106,6 +106,8 @@ private class MemcpyFunction extends ArrayFunction, DataFlowFunction, SideEffect
     not this.hasGlobalName(["bcopy", mempcpy(), "memccpy"]) and
     index = this.getParamDest()
   }
+
+  override TCxxException getExceptionType() { any() }
```

Comment:
Just to make sure I understand. This class now implements `NonThrowingFunction` and override `getExceptionType` to be C++ exceptions. The interpretation being that this class does not throw any C++ exceptions. Is that correct?

Reply:
That is correct. Perhaps thee APi is a bit confusing?

Reply:
Awesome! Thanks!